src/pages/TeamsPage/ExpandedTeams/style.css

```diff
@@ -11,7 +11,7 @@
 }
 
 .paragraph-text {
-    font-family: "Space Mono";
```

Comment:
Should we be setting text styles more globally??

Reply:
It is set, just some styles reset it here and there. I can remove it but it's a carry over from legacy code